libraries/BLE/src/BLEEddystoneTLM.h

```diff
@@ -18,33 +22,34 @@
  */
 class BLEEddystoneTLM {
 public:
-	BLEEddystoneTLM();
-	std::string getData();
-	BLEUUID	 getUUID();
-	uint8_t	 getVersion();
-	uint16_t	getVolt();
-	float	   getTemp();
-	uint32_t	getCount();
-	uint32_t	getTime();
-	std::string toString();
-	void		setData(std::string data);
-	void		setUUID(BLEUUID l_uuid);
-	void		setVersion(uint8_t version);
-	void		setVolt(uint16_t volt);
-	void		setTemp(float temp);
-	void		setCount(uint32_t advCount);
-	void		setTime(uint32_t tmil);
+  BLEEddystoneTLM();
+  std::string getData();
+  BLEUUID   getUUID();
+  uint8_t   getVersion();
+  uint16_t  getVolt();
+  uint16_t  getTemp();
+  float     getFloatTemp();
```

Comment:
In the Original code there was just
`float	   getTemp();`

Now this has changed to `float getFloatTemp();`
and `float getFloatTemp();`

This is a breaking change... 
I don't think that it is a good idea.

Reply:
The suggestion here is to keep the original API `float getTemp();` and use the MACROs for converting temperature types whenever necessary.

Reply:
This should be the final code:
``` cpp
  BLEEddystoneTLM();
  std::string getData();
  BLEUUID   getUUID();
  uint8_t   getVersion();
  uint16_t  getVolt();
  float  getTemp();
```